helpers/use-polling-request.js

```diff
@@ -63,7 +63,10 @@ export default function usePollingRequest({
         try {
           const response = await fetch(`${progressRequestURL}`);
           const data = await response.json();
```

Comment:
you may want to move this to after the `response.status` check. that way `response.text()` or `response.json()` won't potentially error and throw before the status check